src/main/java/org/folio/dew/batch/eholdings/EHoldingsToExportFormatMapper.java

```diff
@@ -30,6 +38,12 @@
 public class EHoldingsToExportFormatMapper {
 
   private static final String ACCESS_TYPE_INCLUDED = "accessTypes";
+  private final SimpleDateFormat dateFormat;
```

Comment:
Maybe it would be better to use another (thread safe) formatter here?

Reply:
What is your sugestion?

Reply:
DateTimeFormatter